lucille-core/src/main/java/com/kmwllc/lucille/core/RunnerManager.java

```diff
@@ -0,0 +1,56 @@
+package com.kmwllc.lucille.core;
+
+import com.kmwllc.lucille.core.Runner;
+import com.kmwllc.lucille.core.Runner.RunType;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.util.concurrent.CompletableFuture;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RunnerManager {
+
+  private static final Logger log = LoggerFactory.getLogger(RunnerManager.class);
+
+  private static volatile RunnerManager instance = new RunnerManager();
+
+  protected static boolean isRunning = false;
+
+  private RunnerManager() {}
+
+  public static RunnerManager getInstance() {
+    return instance;
+  }
+
+  public boolean isRunning() {
+    return isRunning;
```

Comment:
Do we need the isRunning variable?
Could we instead return (future != null && !future.isDone()) here?